recipes/r-n1qn1/LICENSE

```diff
@@ -0,0 +1,506 @@
+
```

Comment:
Is this license not in the source tarball?

Reply:
Found it deep down, missed the `COPYING` file.